src/app/dim-ui/CharacterSelect.test.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import TestRenderer from 'react-test-renderer';
+import CharacterSelect from './CharacterSelect';
+import { DimStore } from '../inventory/store-types';
+import { Frame, Track, ViewPager } from 'react-view-pager';
+import SimpleCharacterTile from 'app/inventory/SimpleCharacterTile';
+
+const stubDimStore: DimStore = {} as DimStore;
+const stubDimStoreArray: DimStore[] = [];
+const dimStore1: DimStore = {} as DimStore;
+const dimStore2: DimStore = {} as DimStore;
+const dimStore3: DimStore = {} as DimStore;
+const mockOnCharacterChanged = jest.fn();
+
+beforeEach(() => {
+  dimStore1.id = '2';
+  dimStore1.isDestiny1 = jest.fn().mockReturnValue(false);
+  dimStore1.isDestiny2 = jest.fn().mockReturnValue(true);
+  dimStore1.isVault = false;
+  dimStore2.id = '3';
+  dimStore2.isDestiny1 = jest.fn().mockReturnValue(false);
+  dimStore2.isDestiny2 = jest.fn().mockReturnValue(true);
+  dimStore2.isVault = true;
+  dimStore3.id = '4';
+  dimStore3.isDestiny1 = jest.fn().mockReturnValue(false);
+  dimStore3.isDestiny2 = jest.fn().mockReturnValue(true);
```

Comment:
This is my ignorance about jest, but why do this instead of just `() => true`?

Reply:
In this situation, they are equivalent. The only reason to use mock functions is if you have tests that care about how many times the function was called or if you want to do something specific. I went ahead and replaced the mocks.